src/js/index.js

```diff
@@ -0,0 +1,36 @@
+import '../css/index.css'
+import { Calc } from './calc.js'
+
+const calculate = new Calc()
+
+const digitList = document.getElementsByClassName('digit')
+const operationList = document.getElementsByClassName('operation')
+const modifier = document.getElementsByClassName('modifier')
+const display = document.getElementById('total')
```

Comment:
```suggestion
const Selectors = {
  digitList: document.getElementsByClassName('digit'),
  operationList: document.getElementsByClassName('operation'),
  modifier: document.getElementsByClassName('modifier'),
  display: document.getElementById('total'),
};
```
이럴게 namespace로 묶어주면 관리하기가 더 쉽지 않을까요!?